container/go/fast_puller.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"flag"
+	"log"
+)
+
+var (
+	name = flag.String("name",
```

Comment:
Get rid of line wrapping here and for every other variable below. The convention in Go is to let lines run as long as necessary and let your editor soft wrap lines for you. See [here](https://github.com/bazelbuild/rules_docker/blob/aafaf340e545322918aac22729ff11638b8c2452/container/go/cmd/extract_config/main.go#L13)